docs/DeveloperGuide.md

```diff
@@ -341,6 +341,70 @@ without breaking existing syntax coloration managed with `TextMate` in `IntelliJ
 </extensions>
 ```
 
+## Language Server API
+
+If you need to `enable/disable` and / or `start/stop` your language server, LSP4IJ provides the LanguageServerManager API which provides this support:
+
+### Enable / Disable 
+
+By default, language server is enabled : in other words, when IntelliJ IDEA starts LSP4IJ tracks the opened files and starts the proper language servers.

```

Comment:
By default, a language server is enabled : in other words, when the IDE starts, LSP4IJ tracks the opened files and starts the matching language servers.

Reply:
fixed